charts/loki-distributed/values.yaml

```diff
@@ -219,7 +219,7 @@ loki:
       storage:
         type: local
         local:
-          directory: /etc/loki/rules
+          directory: {{ if .Values.ruler.sidecar.enabled }}/etc/loki/sc-rules{{ else }}/etc/loki/rules{{ end }}
```

Comment:
If the sidecar is enabled, could it just mount at the exiting path instead of a new path?